include/openssl/nid.h

```diff
@@ -4264,6 +4264,11 @@ extern "C" {
 #define NID_sha3_256 965
 #define OBJ_sha3_256 2L, 16L, 840L, 1L, 101L, 3L, 4L, 2L, 1L
 
+#define SN_sha3_512 "SHA3_512"
```

Comment:
Should this be `SHA3-512`? Also line 4262 looks like a typo?

Reply:
Fixed. Thank you.